test/integration_test.js

```diff
@@ -136,7 +136,7 @@ describe('Integration test on ClientService', async () => {
               {},
           );
 
-          const contractResult = response.result;
+          const contractResult = response.getResult();
```

Comment:
This is a bug from before.
We should always use `getResult` to retrieve the result instead of reading the JavaScript object field directly.

Reply:
This kind of bug fixes should be done in a bugfix PR; otherwise, it's hard to cherry-pick the bug fix.
The node SDK hasn't been used in production yet, so it should be fine for now, but please keep it in mind.